### PR TITLE
[13.0][MIG] report_qweb_pdf_fixed_column: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
The module is still necessary

cc @tecnativa TT23115

### CODE REVIEW

File: report_qweb_pdf_fixed_column/i18n/report_qweb_pdf_fixed_column.pot [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
+"Report-Msgid-Bugs-To: \n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
```
Comment (sergio-teruel) [inline report_qweb_pdf_fixed_column/i18n/report_qweb_pdf_fixed_column.pot:14]: If there is not a translation... We are need this file?

**Review by sergio-teruel — APPROVED** [review]
Only one question... :smile:

File: report_qweb_pdf_fixed_column/i18n/report_qweb_pdf_fixed_column.pot (hunk 1 above)
Comment (pedrobaeza) [reply report_qweb_pdf_fixed_column/i18n/report_qweb_pdf_fixed_column.pot:14]: It's created any way by merge process.

**Review by pedrobaeza — APPROVED** [review]

**Comment by manuelcalerosolis** [comment]
Functional testing ok!

Approved

**Pull request merged**